### ISSUE TITLE
Update regional-planning to specify subLayers

### ISSUE THREAD

**Issue by kellyi** [CoastalResilienceNetwork/GeositeFramework#845]
As part of working on a branch which uses the ArcGIS JS 4.2 API, we'll need to adjust how regional-planning requests its layers. The framework has been updated to make requests using the new API syntax, but it seems like retrieving specific layers might entail specifying which `subLayers` to get from a service:

https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-MapImageLayer.html

This PR has more info on what's currently happening with the layers:

https://github.com/CoastalResilienceNetwork/GeositeFramework/pull/831

TLDR's that the requests are going through but they're not requesting the correct indices for whatever service.
